langs/node.go

```diff
@@ -26,13 +32,59 @@ func (lh *NodeLangHelper) RunFromImage() (string, error) {
 	return "fnproject/node", nil
 }
 
+const funcJsContent = "const fdk = require('@fnproject/fdk') \n\n" +
+	"fdk.handle(function (input, ctx) { \n" +
+	"  return 'Hello ' + JSON.stringify(input) \n" +
+	"}) \n"
+
+const packageJsonContent = "{\n" +
+	"   \"name\": \"hellofn\", \n" +
+	"   \"version\": \"1.0.0\", \n" +
+	"   \"description\": \"example function\",\n" +
+	"   \"main\": \"func.js\",\n" +
+	"   \"author\": \"\",\n" +
+	"   \"license\": \"Apache-2.0\", \n" +
+	"   \"dependencies\": { \n" +
+	"      \"@fnproject/fdk\": \"0.x\"\n " +
```

Comment:
Maybe put here a valid version?

Reply:
That is a valid qualifier  and if you run npm install out side of the function it will lock to the version it got. 

We found with java that having to bump the CLI for every version caused more problems then having the version vary of its own free will. 

In an ideal world i would have had time to query npm for the latest version to fix (which gives you the best of both worlds) but didn't have time.

Reply:
Okay, just tried that, you are right, it works.